google/cloud/forseti/common/gcp_api/storage.py

```diff
@@ -562,6 +562,14 @@ def get_bucket_acls(self, bucket, user_project=None):
                                 'retrying.', bucket)
                     return self.get_bucket_acls(bucket, self._user_project)
 
+            raw_error = e.args[1]
+            error = raw_error.decode('utf-8')
+            formatted_error = json.loads(error)
+            error_code = formatted_error['error']['code']
+            if error_code == 400:
```

Comment:
Checking for 400 would only handle the case where the Forseti server service account has the Viewer role, but not case where the SA does not have that role. Can we try to instead not make the call if the storage object has uniform bucket level access enabled? It is part of the [CAI/API resource](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource-representations).